app/assets/javascripts/sufia/app.js

```diff
@@ -12,6 +12,8 @@ Sufia = {
   saveWorkControl: function() {
     var sw = require('sufia/save_work/save_work_control');
     new sw.SaveWorkControl($("#form-progress")).activate();
+      FixedSticky.tests.sticky = false;
+      $('#fixedwidget').fixedsticky();
```

Comment:
On second thought, this should not go into `save_work_control.es6`, but it should be within a new method in this file.